config/software/libxslt.rb

```diff
@@ -49,7 +55,7 @@
   # iteration treats colons as a delimiter so we are using a cygwin
   # style path to accomodate
   configure_commands = [
-    "--with-libxml-prefix=#{install_dir.sub("C:", "/C")}/embedded",
+    "--with-libxml-lib-prefix=#{install_dir.sub("C:", "/C")}/embedded/lib/pkg-config",
```

Comment:
I know you said the older versions of libxml2 and libxslt weren't building successfully but we've somehow been using them without problems in a number of our omnibus packaged products. It worries me that we suddenly need to replace `--with-libxml-prefix` with `--with-libxml-lib-prefix` and use a different path. I am going to look into this to understand this better before approving it.